examples/py310/doc-examples-python-apis.py

```diff
@@ -160,12 +163,90 @@ async def example_API_TopicPublish(topic_client: TopicClientAsync):
     match response:
         case TopicPublish.Success():
             print("Successfully published a message")
-        case TopicPublish.Error():
-            print("Error publishing a message: ", response.message)
+        case TopicPublish.Error() as error:
+            print("Error publishing a message: ", error.message)
```

Comment:
For consistency, can you use f strings for the error print statements? 🐍

Reply:
Done!